### DISCUSSION TITLE
dubbo async  invoke

### DISCUSSION THREAD

**Discussion by newboy2004** [apache/skywalking#8082] [Q&A]
dubbo use aync invoke, skywalking  chain trace can break.
![Uploading image.png…]()
please ask,how does slove this problem?tks

**Comment by wu-sheng** [comment]
I can not see what you uploaded. But AFAIK, we don't support `async invoke` for dubbo plugin.
No one contributed some enhancement about this yet.

**Reply by newboy2004** [reply]
sorry,i can not  you answer tks!!!
